lib/inputs/home_input.dart

```diff
@@ -107,249 +114,204 @@ class _HomeInputForm extends State<HomeInputForm> {
-                ],
-              ),
-            ],
-          ),
-          // Submit Button
-          Row(
-            mainAxisAlignment: MainAxisAlignment.center,
-            crossAxisAlignment: CrossAxisAlignment.center,
-            children: <Widget>[
-              Column(
-                children: <Widget>[
-                  RaisedButton(
-                    onPressed: () {
-                      // Validate returns true if the form is valid, or false
-                      // otherwise.
-                      if (_formKey.currentState.validate()) {
-                        // If the form is valid, display a Snackbar.
-                        Scaffold.of(context).showSnackBar(
-                            SnackBar(content: Text('Processing Data')));
-                      }
-                      FormState form = _formKey.currentState;
-                      form.save();
-                      int calculatedEmission = calculateEmission(
-                          newHomeInput.billingCycle,
-                          newHomeInput.kwh,
-                          newHomeInput.jurisdiction);
-                      // Add lines below after DB implementation for Fields used
-                      // -------------------------------------------------------
-                      //addEntry(
-                      //    calculatedEmission, DateTime.now(), 'Home Energy');
-                      // -------------------------------------------------------
-                      _showDialog(calculatedEmission);
-                    },
-                    child: Text('Submit'),
+                  // Input
+
+                  Row(
+                    /// Submit button
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    mainAxisSize: MainAxisSize.max,
+                    children: <Widget>[
+                      Container(
+                        padding: EdgeInsets.only(top: 12),
+                        child: RaisedButton(
+                          color: CustomMaterialColor.emphasisColor,
+                          onPressed: () {
+                            // Validate returns true if the form is valid, or false
+                            // otherwise.
+
+                            if (_formKey.currentState.validate()) {
+                              // If the form is valid, display a Snackbar.
+                            }
+                            FormState form = _formKey.currentState;
+                            form.save();
+
+                            int calculatedEmission = calculateEmission(
+                                newHomeInput.billingCycle,
+                                newHomeInput.kwh,
+                                newHomeInput.jurisdiction);
+                            // Add lines below after DB implementation for Fields used
+                            // -------------------------------------------------------
+                            //addEntry(
```

Comment:
Same goes for this, it should work.
Can you uncomment in and change the conversion to grams.